tests/integration/mutation/update/crdt/pncounter_test.go

```diff
@@ -83,10 +69,12 @@ func TestMutationUpdate_PNCounterInt_NoError(t *testing.T) {
 }
 
 // This test documents what happens when an overflow occurs in a PN Counter with Int type.
-// In this case the value rolls over to the minimum int64 value.
-func TestMutationUpdate_PNCounterIntWithOverflow_NoError(t *testing.T) {
+// Note: This documents a sub optimal behaviour of the system due to json unmarshalling
+// MaxInt64 into a float which loses precision and converting it back to int64 sets it as a negative
+// value.
+func TestPNCounterUpdate_IntKindWithPositiveIncrementOverflow_RollsOverToMinInt64(t *testing.T) {
```

Comment:
praise: I even misinterpreted the behaviour by reading the code. With this new name it's very transparent what the test does. Thanks!